src/client.js

```diff
@@ -158,7 +160,17 @@ export default class Client {
     });
   }
 
-  fetchCollection(id, query = collectionQuery()) {
+  fetchCollection(id, query = collectionNodeQuery()) {
```

Comment:
Should the queries really vary if they're being requested off node, or off of another object? Should we maybe make an abstraction around fragments, and then share common code between fragments and queries for types?